src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -196,10 +197,8 @@ export class SbbFormField implements ComponentInterface {
     if (labels.length > 1 && createdLabel) {
       createdLabel.remove();
     }
-    const inputId = this._input?.id;
-    if (inputId) {
-      labels.forEach((l) => (l.htmlFor = inputId));
-    }
+    this._labels = Array.from(this._element.querySelectorAll('label'));
```

Comment:
This assumes that label is slotted inside the component, but it can be anywhere in the light DOM?
```
<label for="select-1">Label</label>
<sbb-form-field>
  <sbb-select id="select-1"></sbb-select>
</sbb-form-field>
```

or more general (not form field related I know)
```
<label for="select-1">Label</label>
<sbb-select id="select-1"></sbb-select>
```

Reply:
Needs documentation